src/Schema/Values/OwnerType.ts

```diff
@@ -72,6 +74,8 @@ export type ScreenOwnerType =
   | OwnerType.collection
   | OwnerType.consignmentSubmission
   | OwnerType.consign
+  | OwnerType.conversation
```

Comment:
I notice we have [`inboxConversation`](https://github.com/artsy/cohesion/blob/e6a7e2cbfec2e61151caff87224078e26a4ae131/src/Schema/Values/OwnerType.ts#L80) as an owner type. I'm not sure what the difference is but want to raise it.

Reply:
Great question! @anipetrov and I discussed `inboxConversation` as well. We decided that ideally this would have been `conversation` from the beginning and it's only being used as the destination for the `tappedInboxConversation` event right now. Since we are now starting to implement events on the actual `conversation` screen, we decided that it's okay to switch to `conversation` moving forward.

Let me know if that's okay!

Reply:
Yeah that makes sense to me! Thanks for the context! 👍